controllers/metal3.io/host_state_machine.go

```diff
@@ -301,7 +301,8 @@ func (hsm *hostStateMachine) handleRegistering(info *reconcileInfo) actionResult
 }
 
 func (hsm *hostStateMachine) handleInspecting(info *reconcileInfo) actionResult {
-	actResult := hsm.Reconciler.actionInspecting(hsm.Provisioner, info)
+	refresh := hasInspectAnnotation(hsm.Host)
```

Comment:
This can be moved inside `actionInspecting`

Reply:
Done.